tutorials/sigfox-gw/index.md

```diff
@@ -193,21 +192,21 @@ The integration uses the Sigfox API V2 to automate configuring the callbacks. In
 
         (venv) $ cd ..
 
-1. Generate a Python client library from the Sigfox API V2 definition:
+1. Check that you have docker running, by executing for example:
 
-        curl -X POST -H "content-type:application/json" -d '{"swaggerUrl":"https://support.sigfox.com/api/apidocs"}' https://generator.swagger.io/api/gen/clients/python
+        (venv) $ docker ps
 
-1. The command will output a URL to download the generated client library. Copy the URL and download the file with a command similar to this:
+1. Next we will generate the Python client libraries from the [Sigfox API definition](https://support.sigfox.com/api/apidocs). To do this, we have to compile and use a specific version of [swagger-codegen](https://github.com/swagger-api/swagger-codegen). Execute the following command to setup an environment variable for the swagger-codegen version tag to be used:
 
-        (venv) $ curl -o python_client.zip https://generator.swagger.io/api/gen/download/[YOUR_UNIQUE_STRING]
+        (venv) $ version=3.0.11
 
-1. Unzip the downloaded client library:
+1. The following command will use docker to first download the swaggerapi/swagger-codegen-cli image, and then use the code generator's specific version, to compile a Python client library from the Sigfox API definition:
 
-        (venv) $ unzip python_client.zip
+        (venv) $  docker run --rm -v ${PWD}:/local swaggerapi/swagger-codegen-cli-v3:$version generate -i https://support.sigfox.com/api/apidocs -l python -o /local/out/python-$version
```

Comment:
instead of ${PWD}, use "$(pwd)", so you don't have to rely on the shell for setting that variable beforehand